plugins/services/src/js/service-configuration/ServiceStorageConfigSection.js

```diff
@@ -31,160 +28,140 @@ class ServiceStorageConfigSection extends ServiceConfigBaseSectionDisplay {
     return "CreateService:ServiceConfigDisplay:App:Storage";
   }
 
+  getVolumeSizeValue(value, type = null) {
+    if (value == null) {
+      return getDisplayValue(value);
+    }
+
+    if (type === VolumeConstants.type.external) {
+      // External volumes specify size in GiB
+      value = value * 1024;
+    }
+
+    return formatResource("disk", value);
+  }
+
   /**
    * @override
    */
   getDefinition() {
-    const { onEditClick } = this.props;
+    const { appConfig } = this.props;
 
-    return {
+    if (
```

Comment:
Not sure how I feel about this `sanity check` but for safety, I don't want to continue when there is no data to go on that way avoiding any error throw.